src/main/java/com/hms_networks/americas/sc/extensions/historicaldata/HistoricalDataQueueManager.java

```diff
@@ -93,6 +93,18 @@ public class HistoricalDataQueueManager {
    */
   private static boolean lastReadDataPointsEmpty = false;
 
+  /** Should EBD requests use last point timestamp. */
```

Comment:
Suggestion: `Boolean indicating whether EBD requests should use the last data point's timestamp.`

Reply:
changed